ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/EmailIntegrationForm.tsx

```diff
@@ -455,6 +457,20 @@ function EmailIntegrationForm({
                             ))}
                         </SelectSingle>
                     </FormLabelGroup>
+                    <FormLabelGroup label="" fieldId="notifier.email.skipTLSVerify" errors={errors}>
+                        <Checkbox
+                            label="Skip TLS verification"
+                            id="notifier.email.skipTLSVerify"
+                            isChecked={values.notifier.email.skipTLSVerify}
+                            onBlur={handleBlur}
+                            onChange={(event, value) => onChange(value, event)}
+                            isDisabled={
+                                !isEditable ||
+                                (values.notifier.email.disableTLS &&
+                                    values.notifier.email.startTLSAuthMethod === 'DISABLED')
```

Comment:
@clickboo @vjwilson What is the relationship in backend behavior and frontend interaction?
* If TLS is disabled, `skipTLSVerify` is redundant, and its `true` or `false` value is ignored?
* If not redundant and ignored, might this condition block user from being able to change the value?

Reply:
Point 1 above is true. If TLS is disabled, and STARTTLS == disabled, skipTLSVerify is redundant.